verilog/systemverilog/Includes.g4

```diff
@@ -1,7 +1,7 @@
 // Author: Mustafa Said Ağca
 // License: MIT
 
-lexer grammar SystemVerilogLexer;
+grammar Includes;
```

Comment:
Why just don't use the common structure with `lexer` and `parser` grammars that we have in other grammars? With `tokenVocab` but without `import`.

Reply:
There's a choice here with either making the grammar split or combined. The problem is that the current situation has elements of both, but it's done wrong for either. There are many string literals in the parser rules in the "parser grammar", but these are not actually noted in the "lexer grammar". To make this a split grammar, I'd have to enter all the string literals into the lexer, and I just want to get this to work in under a minute of work--as I said, I don't have a lot of time working on scraping and rewriting the C++ grammar. Since there's another grammar that will replace this grammar, I felt this an okay decision.